variables.tf

```diff
@@ -67,10 +67,11 @@ variable "users" {
 
 variable "vpc_endpoint" {
   type = object({
-    security_group_ids     = list(string)
-    address_allocation_ids = list(string)
-    subnet_ids             = list(string)
-    vpc_id                 = string
+    address_allocation_ids = optional(list(string))
+    security_group_ids     = optional(list(string))
+    subnet_ids             = optional(list(string))
+    vpc_endpoint_id        = optional(string)
```

Comment:
It would be cool to add some validation since you can only use vpc_endpoint_id when setting endpoint_type to VPC_ENDPOINT, but I leave that up to you no hard requirement for me: https://github.com/schubergphilis/terraform-aws-mcaf-aurora/blob/master/variables.tf#L61

Reply:
Isn't this already done (forced) by this:  https://github.com/sbpdvb/terraform-aws-mcaf-transfer/blob/feature/add-vpc-endpoint-id-support/main.tf#L55. If you use the endpoint_details block.

Reply:
This is not really providing feedback to the user while validation does.